conf/scylla.yaml

```diff
@@ -258,9 +258,10 @@ batch_size_fail_threshold_in_kb: 1024
 # be set.
 # broadcast_rpc_address: 1.2.3.4
 
+enable_user_defined_functions: true
 # Uncomment to enable experimental features
-# experimental_features:
-#     - udf
+experimental_features:
+    - udf
```

Comment:
Why this?

Reply:
This will be removed when PR will be completed. I needed to rebranch and thus commited this file.